docs/backend/SYCL.md

```diff
@@ -310,12 +310,14 @@ export CPLUS_INCLUDE_DIR=/path/to/oneMKL/buildWithCublas/include:$CPLUS_INCLUDE_
 export CPLUS_INCLUDE_DIR=/path/to/oneMKL/include:$CPLUS_INCLUDE_DIR
 
 # Build LLAMA with Nvidia BLAS acceleration through SYCL
+# Setting GGML_SYCL_ARCH is optional but can improve performance
+GGML_SYCL_ARCH=sm_80 # Example architecture
```

Comment:
how about extract other vendors' detail to another file and leave a link here? It will be quite confusing if mixing so much archs

Reply:
Which other file would you suggest? As far as I can tell all the SYCL documentation is in this file. I don't see how else I could document this without a big documentation refactor.
The option only allows setting one architecture and `sm_80` is just an example. The link to the list of available architecture is linked below if this is what you are worried about: https://github.com/ggerganov/llama.cpp/pull/10266/files#diff-4f40b10d65d19565440cfa25acd0129a87a596de19c2298991021a9a01a040e6R650